auth/src/auth.cc

```diff
@@ -68,6 +68,7 @@ Auth* Auth::GetAuth(App* app, InitResult* init_result_out) {
   Auth* existing_auth = FindAuth(app);
   if (existing_auth) {
     if (init_result_out != nullptr) *init_result_out = kInitResultSuccess;
+    LogHeartbeat(existing_auth);
```

Comment:
Recommend to add a comment above about why we do this here.
Something like:
"Log heartbeat data when instance getters are called (go/firebase-platform-logging-design)"

Reply:
Done.